quiz_portal/settings/base.py

```diff
@@ -135,6 +135,7 @@
 CKEDITOR_RESTRICT_BY_USER = True
 CKEDITOR_ALLOW_NONIMAGE_FILES = False
 
-RECAPTCHA_PRIVATE_KEY = os.environ['RECAPTCHA_PRIVATE_KEY']
-RECAPTCHA_PUBLIC_KEY = os.environ['RECAPTCHA_PUBLIC_KEY']
+RECAPTCHA_PRIVATE_KEY = os.environ.get('GOOGLE_RECAPTCHA_SECRET_KEY')
```

Comment:
why changed the name?

Reply:
Sir, I saved my previous ReCaptcha using this name only. No problem I will change it.

Reply:
Done